fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+        description: The longitude of the shipping address.
+      name:
+        type: string
+        description: The full name of the person associated with the payment method.
+      phone:
+        type: string
+        description: The phone number at the shipping address.
+      province:
+        type: string
+        description: The name of the state or province of the shipping address.
+      province_code:
+        type: string
+        description: The two-letter abbreviation of the state or province of the shipping address.
+      zip:
+        type: string
+        format: zip
+        description: The zip or postal code of the shipping address.
+        
+  
+    
+  billing_address:
+    type: object
+    description: The mailing address associated with the payment method. This address is an optional field that will not be available on orders that do not require one.
+    properties:
+      address1:
+        type: string
+        description: The street address of the billing address.
+      address2: 
+        type: string
+        description: An optional additional field for the street address of the billing address.
+      city:
+        type: string
+        description: The city of the billing address.
+        
+      company:
+        type: string
+        description: The company of the person associated with the billing address.  
+        
+      country:
+        type: string
+        description: The name of the country of the billing address.
+      
+      country_code:
+        type: string
+        description: The two-letter code (ISO 3166-1 alpha-2 two-letter country code) for the country of the billing address.
+      
+      first_name:
+        type: string
+        description: The first name of the person associated with the payment method.
+      last_name:
+        type: string
+        description: The last name of the person associated with the payment method.
+      latitude:
+        type: string
+        format: latitude
+        description: The latitude of the billing address.
+      longitude:
+        type: string
+        format: longitude
+        description: The longitude of the billing address. 
+      name:
+        type: string
+        description: The full name of the person associated with the payment method.
+      phone:
+        type: string
+        description: The phone number at the billing address.
+      province:
+        type: string
+        description: The name of the state or province of the billing address.
+      province_code:
+        type: string
+        description: The two-letter abbreviation of the state or province of the billing address.
+      zip:
+        type: string
+        format: zip
+        description: The zip or postal code of the billing address.
+        
+        
+      
+
+  Error:
+    type: object
+    description: |  
+      Object describing errors (values) on specific attributes (keys) that prevented a draft order from being saved.
```

Comment:
Object describing errors (values) on specific attributes (keys) that prevented an operation from being completed successfully.